saliency/xrai_test.py

```diff
@@ -180,6 +180,32 @@ def testBaselines(self):
     self.assertTrue(np.array_equal(baseline_2, calls[1][2]['x_baseline']),
                     msg='IG was called with incorrect baseline.')
 
+  def testBaseAttribution(self):
+    base_attribution = np.random.rand(IMAGE_SIZE, IMAGE_SIZE, 3)
+
+    # Calculate XRAI attribution using GetMask(...) method.
+    heatmap = self.xrai.GetMask(x_value=self.input_image,
+                                base_attribution=base_attribution)
+    # Verify the result.
```

Comment:
maybe expand the comment a bit here, that you know there is conservation law (the sum needs to match)

Reply:
Done.